control-plane/CMakeLists.txt

```diff
@@ -83,19 +83,25 @@ set (CONTROLPLANE_SOURCES
   ${CONTROLPLANE_SRCS}
   )
 
-set (CONTROLPLANE_HDRS
+set (CONTROLPLANE_SRCS
```

Comment:
I don't understand this change. `CONTROLPLANE_SRCS` is defined above already.

Reply:
cmake is running top to down in the file.  There is no further use of `CONTROLPLANE_SRCS` except for cpplint, so it should be fine.

However, it confuses code.  I  will change code.